Source/SGDBMetadataSettingsView.xaml.cs

```diff
@@ -23,23 +11,23 @@ public SGDBMetadataSettingsView()
             
             cmbDimensions.ItemsSource = new List<string>
             {
-                "any",
+                "Any",
                 "460x215",
                 "920x430",
                 "600x900",
                 "342x482",
-                "legacy"
+                "Legacy"
             };
             cmbDimensions.SelectedIndex = 0;
 
             cmbStyles.ItemsSource = new List<string>
             {
-                "any",
-                "alternate",
-                "blurred",
-                "white_logo",
-                "material",
-                "no_logo"
+                "Any",
+                "Alternate",
+                "Blurred",
+                "White Logo",
+                "Material",
+                "No Logo"
```

Comment:
The strings in this list are the direct references used for api requests, postman cares about case sensitivty.  We can use a dictionary for key value pairs.

Reply:
Ah, that was an oversight of mine. However, the file you're referencing in this is a UI file. The highlighted strings are _only_ for display in the interface. These need to be changed in `SGDBMetadataSettings.cs` instead. I have corrected this. Fixed and pushed.